python/private/pypi/whl_config_repository.bzl

```diff
@@ -0,0 +1,46 @@
+"""whl_config_library implementation for WORKSPACE setups."""
```

Comment:
NOTE: we could in theory reuse the `hub_repository.bzl` which is only used in `bzlmod` in here, but I want to leave this out of scope of this PR.

The main realization of this PR is that the WORKSPACE to bzlmod mapping of components is as following:
- bzlmod extension maps to the `pip_parse` repository rule
- the bzlmod hub repo maps to the `config` repository created here
- the spokes in both are the same.

Reply:
SGTM.

Yeah, without some more broad refactorings, re-using some of the "common" code between workspace and bzlmod is somewhat convoluted.